pandas/compat/pyarrow.py

```diff
@@ -12,6 +12,7 @@
     pa_version_under3p0 = _palv < Version("3.0.0")
     pa_version_under4p0 = _palv < Version("4.0.0")
     pa_version_under5p0 = _palv < Version("5.0.0")
+    pa_version_under6p0 = _palv < Version("6.0.0")
```

Comment:
I think also need to add `pa_version_under6p0 = True` below

Reply:
Ah, yes, thanks